seekinternational/seek-asia-style-guide
# feat(JobCard): add more track link type and refactoring

Related to #311
Add company, jobTitle and location to `trackLinkClicked`.
Refactoring

## How to test
Go to JobCard, checked `Custom Link` and `Job Title Component`
1. Click company name and check console.log
2. Click job title and check console.log
3. Click location and check console.log
